jboss/container/wildfly/launch/https/module.yaml

```diff
@@ -1,20 +0,0 @@
-schema_version: 1
-name: jboss.container.wildfly.launch.https
-version: '1.0'
-description: Configures legacy SSL
-execute:
-  - script: configure.sh
-    user: '185'
-envs:
-  - name: "HTTPS_PASSWORD"
```

Comment:
We are removing the uses of these env variables, but, are we going to supply any alternative to configure SSL via Elytron?

Reply:
This script already handles SSL configuration using elytron, right? : https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/elytron/module.yaml#L20

Reply:
Correct, the alternative is already there at elytron.sh script. So, forget my comment.